src/Microsoft.ApplicationInsights.AspNet/ApplicationInsightsExceptionMiddleware.cs

```diff
@@ -26,7 +27,9 @@ public ApplicationInsightsExceptionMiddleware(RequestDelegate next, TelemetryCli
             {
                 if (this.telemetryClient != null)
```

Comment:
This condition will never be false. If the TelemetryClient service is not available, construction of this object will fail and the Invoke method will not be called. I'd like to remove the if statement. Any objections?